openapi.yml

```diff
@@ -431,11 +431,11 @@ paths:
                     type: object
                     properties:
                       level:
-                        type: string
+                        type: integer
```

Comment:
Any reason they're Integer here and Number below?

Reply:
No particular reason, as it would be the same either way. Updated to use number instead